src/DynamoCoreWpf/Services/IconServices.cs

```diff
@@ -80,8 +81,20 @@ internal ImageSource LoadIconInternal(string iconKey)
             ResourceManager rm = new ResourceManager(assemblyName + imagesSuffix, resourceAssembly);
 
             ImageSource bitmapSource = null;
+            Bitmap source = null;
+
+            try
+            {
+                using (var ms = new MemoryStream(rm.GetObject(iconKey) as byte[]))
+                {
+                    source = new Bitmap(ms);
+                }
+            }
+            catch(Exception e)
+            {
+                //log the error
```

Comment:
This is where we can support some logging or alternative loading option for legacy packages